gammapy/stats/counts_statistic.py

```diff
@@ -217,14 +217,6 @@ def __init__(self, n_on, n_off, alpha, mu_sig=None):
         else:
             self.mu_sig = np.asanyarray(mu_sig)
 
-    @property
-    def mu_bkg(self):
```

Comment:
I agree this can be removed, in the case of "counts statistics", we expect `n_bkg = mu_bkg`, `n_sig = mu_sig`, `n_on = mu_on` etc. anyway. However ever I noticed that` n_bkg = mu_bkg` is not fulfilled (which could be considered as the actual bug...). This is because we deliberately set `mu_sig = 0` in `WStatCountsStatistic.__init__`, however in the case of no known signal component it should be rather set to `n_on - alpha * n_off`. Maybe we should fix this as well? Of possibly rename `mu_sig` here completely, as it does not correspond to the what is defined [here](https://docs.gammapy.org/0.18.1/stats/index.html#notations)...